.eslintrc.yaml

```diff
@@ -16,6 +16,9 @@ extends:
   - plugin:import/typescript
   - plugin:prettier/recommended
   - prettier # Removes eslint rules that conflict with prettier.
+  # Recommended by jest-playwright
+  # https://github.com/playwright-community/jest-playwright#globals
+  - plugin:jest-playwright/recommended
```

Comment:
I'm not sure if this has to go at the end - because usually Prettier recommends their ruleset to be the last one, afaik

Reply:
Ah, got it. I'll reorder these!